tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "outDir": "./lib",
     "target": "ESNext",
     "moduleResolution": "node",
-    "module": "CommonJS"
+    "module": "CommonJS",
+    "esModuleInterop":true
```

Comment:
This change makes it easier to consume libraries from npm - without this we had to do some ugly jumps in code to satisfy TS + ensure CJS compatibility in our output. Didn't feel worth fighting.

Reply:
ack